scripts/us_census/pep/population_estimate_by_race/preprocess.py

```diff
@@ -0,0 +1,936 @@
+        +df['Black female']+df['Other races male']+df['Other races female']
+    df['White Alone'] = df['White male'] + df['White female']
+    df['Black or African American Alone'] = df['Black male'] + \
+        df['Black female']
+    df.drop(columns=['Info','level_0','White male','White female','Black male'\
+        ,'Black female','Other races male','Other races female'],inplace=True)
+    final_df = pd.DataFrame()
+    final_df = pd.concat([final_df, df])
+    df['geo_ID'] = df['geo_ID'].str[:-3]
+    df = df.groupby(['Year','geo_ID']).sum().\
+        reset_index()
+    df.drop(columns=['Total'], inplace=True)
+    df.columns = df.columns.str.replace('White Alone',
+                                        'Count_Person_WhiteAlone')
+    df.columns = df.columns.str.replace('Black or African American Alone',\
+            'Count_Person_BlackOrAfricanAmericanAlone')
+    df_temp = pd.DataFrame()
+    df_temp = pd.concat([df, df_temp])
+    df_temp['geo_ID'] = "country/USA"
+    df_temp = df_temp.groupby(['Year','geo_ID']).sum().\
+        reset_index()
+    df = pd.concat([df, df_temp])
+    df.to_csv("output/USA_Population_Count_by_Race_National_State_1980.csv")
+    return final_df
+
+
+def _clean_xls2_file(df: pd.DataFrame) -> pd.DataFrame:
+    """
+    This method cleans the dataframe loaded from a xls file format.
+    Also, Performs transformations on the data.
+
+    Arguments:
+        df (DataFrame) : DataFrame of xls dataset
+
+    Returns:
+        df (DataFrame) : Transformed DataFrame for xls dataset.
+    """
+    df['Race'] = (df['Race/Sex Indicator'].str.replace(" female", ""))
+    df['Race'] = (df['Race'].str.replace(" male", ""))
+    df = df.drop(['Race/Sex Indicator'], axis=1)
+    for col in df.columns:
+        df[col] = df[col].astype(str)
+        df[col] = df[col].str.replace(",", "")
+    extras = ['Year of Estimate', 'FIPS State Code', 'State Name', 'Race']
+    cols = df.columns.drop(extras)
+    df[cols] = df[cols].apply(pd.to_numeric, errors='coerce')
+    # All the age groups are being added up to get total value.
+    df['count']=df['Under 5 years']+df['5 to 9 years']+df['10 to 14 years']\
+        +df['15 to 19 years']+df['20 to 24 years']+df['25 to 29 years']+\
+        df['30 to 34 years']+df['35 to 39 years'] +df['40 to 44 years']	\
+        +df['45 to 49 years'] +df['50 to 54 years'] +df['55 to 59 years']\
+        +df['60 to 64 years'] +df['65 to 69 years'] +df['70 to 74 years']\
+        +df['75 to 79 years'] +df['80 to 84 years'] +df['85 years and over']
+    df = df.drop(['Under 5 years','5 to 9 years','10 to 14 years',\
+        '15 to 19 years','20 to 24 years','25 to 29 years',\
+        '30 to 34 years','35 to 39 years','40 to 44 years',\
+        '45 to 49 years','50 to 54 years','55 to 59 years',\
+        '60 to 64 years','65 to 69 years','70 to 74 years','75 to 79 years',\
+        '80 to 84 years','85 years and over'],axis=1)
```

Comment:
Use list and loop - for adding.
list - for dropping.

Reply:
used loop to add the ages and dropped the list